src/components/navigation-header/index.tsx

```diff
@@ -50,50 +48,33 @@ const MobileMenuButton = () => {
  * the elements with CSS on tablet and smaller viewports.
  */
 const AuthenticationControls = () => {
-	const { showAuthenticatedUI, signIn, signOut, user } = useAuthentication()
-
-	/**
-	 * @TODO implement loading skeleton
-	 * https://app.asana.com/0/1202097197789424/1202791375540720/f
-	 *
-	 * The `showUnauthenticatedUI` boolean is not used here because we want to
-	 * show the sign in/up elements by default until loading skeletons are
-	 * implemented. Most of our users do not have Learn accounts, so our goal is
-	 * to remove the flash for the majority of a users as a temporary effort.
-	 */
+	const { signIn, signOut, user } = useAuthentication()
 
-	let content
-	if (showAuthenticatedUI) {
```

Comment:
❓ checking to make sure we're not using this for the bookmark or progress stuff.